tensorbay/apps/__init__.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+"""apps module."""
+
+from tensorbay.apps.sextant import Benchmark, Evaluation, SextantClient
```

Comment:
So the usage is:
```python
from tensorbay.apps import Benchmark
```
which is not good for me, because if there are multiple apps here. All class in all apps will be all list under tensorbay.apps.
I recommend to separate different apps into different module.